src/iab.ts

```diff
@@ -91,6 +91,97 @@ export const TcfV2VendorList = t.intersection([
 ]);
 
 /**
- * Overload StoredContentfulEntry as a type
+ * Overload TcfV2VendorList as a type
  */
 export type TcfV2VendorList = t.TypeOf<typeof TcfV2VendorList>;
+
+/**
+ * TCF GVL v3 purpose configuration
+ */
+export const TcfGvlV3Purpose = t.type({
+  id: t.number,
+  name: t.string,
+  description: t.string,
+  illustrations: t.array(t.string),
+});
+
+/**
+ * Type override
+ */
+export type TcfGvlV3Purpose = t.TypeOf<typeof TcfGvlV3Purpose>;
+
+/**
+ * TCF GVL v3 data categories configuration
+ */
+export const TcfGvlV3DataCategory = t.type({
+  id: t.number,
+  name: t.string,
+  description: t.string,
+});
+
+/**
+ * Type override
+ */
+export type TcfGvlV3DataCategory = t.TypeOf<typeof TcfGvlV3DataCategory>;
+
+/**
+ * TCF GVL v3 stack configuration
+ */
+export const TcfGvlV3Vendor = t.intersection([
```

Comment:
I'm wondering if this is a superset of TcfVendor. It looks like it is, though I may be missing something. If so, it might make sense to include TcfVendor in the intersection instead of redefining id, name, purposes, etc. It would make sense if it is a superset because that would make v2 and v3 compatible with each other.

Reply:
There is a lot of overlap between TcfVendor and TcfGvlV3Vendor, but there is the addition of multiple fields and the update of a couple previously optional fields to required, but going along with my previous comment, I was thinking about just chucking the old version out when we totally transition to the new version